### DISCUSSION TITLE
「高危委托」攻坚战：与空洞为敌

### DISCUSSION THREAD

**Discussion by Phospherus9484** [share121/inter-knot#2374] [其他]
![retouch_2024081611305350](https://github.com/user-attachments/assets/b6ead324-755b-4645-adb4-b055e0e37e1d)

空洞活性大幅提升，已开始高级别警戒!
我是受雇于黑曜石营的佣兵团团长，因空洞活性已达到预警值，黑曜石营现已采取最强硬的镇压方针
无差别消灭空洞内一切活物。
无论是以骸，劫匪，还是无资质的佣兵，只要涉足活性区，都需被歼灭。
我们即将把蛮横力士缉拿归案时，突然出现一个看似以骸，又像是受到侵蚀的人类……
三队全员员均不敌，惨死在空洞里，只剩我一个人逃了出来……
我知道这个委托有些不近人情，但我也是收钱办事…
绳匠请注意安全！

**Comment by Genshin-Neuvillette** [comment]
阿米诺斯，那个黑化鼠女差点让我永远留在空洞
